shared/domains/chelonia/db.js

```diff
@@ -165,11 +169,16 @@ export default ((sbp('sbp/selectors/register', {
         }
       }
       await sbp('chelonia.db/set', entry.hash(), entry.serialize())
-      await sbp('chelonia.db/set', getLogHead(contractID), JSON.stringify({ HEAD: entry.hash(), height: entry.height() }))
+      await sbp('chelonia.db/set', getLogHead(contractID), JSON.stringify({
+        HEAD: entry.hash(),
+        previousKeyOp: entry.isKeyOp() ? entry.hash() : entry.previousKeyOp(),
+        height: entry.height()
+      }))
       console.debug(`[chelonia.db] HEAD for ${contractID} updated to:`, entry.hash())
       await sbp('chelonia/db/setEntryMeta', contractID, entryHeight, {
         hash: entry.hash(),
-        date: new Date().toISOString()
+        date: new Date().toISOString(),
+        ...(entry.isKeyOp() && { isKeyOp: true })
```

Comment:
Could you add a comment above line 178 for what each of these fields is used for?

I.e. hash is used to map the height to the hash in order to make forward streaming possible, date is used to map the height to the date in order to make search easier, and `isKeyOp` is used for...?